tests/compute/test_transform.py

```diff
@@ -682,17 +682,56 @@ def _check(g, new_g, induced_nodes):
 
 @parametrize_dtype
 def test_to_simple(index_dtype):
```

Comment:
Does the function support GPU? If not, could you add a unittest skip?

Reply:
No.
Let me add skip test